assets/templates/status_entry.html

```diff
@@ -1,5 +1,14 @@
+<script type="text/javascript">
+$( document ).ready(function() {
+  $.get('http://crossorigin.me/http://ip-api.com/json/{{pubIP}}?fields=3', func$
```

Comment:
This file does look corrupted to me, also use https://freegeoip.net/json/ to obtain the country instead